Framework/PythonInterface/mantid/plots/datafunctions.py

```diff
@@ -401,15 +401,17 @@ def get_bins(workspace, wkspIndex, withDy=False):
 
     """
     bins = get_bin_indices(workspace)
-    y_values = []
```

Comment:
Could you help me understand this a little better? Perhaps the naming was not great before.

`readY` gives back the signal values for a given row of a 2D workspace but we seem to be accessing the row by bin index?

Reply:
👍 Good question, I think this is some dodgy naming which is quite confusing. I think `bin_index` should probably be named `workspace_index` or `row_index`. And `wkspIndex` should probably be called `bin_index`